LondonDataServices.IDecide.Portal.Tests.Acceptance/appsettings.Acceptance.json

```diff
@@ -0,0 +1,84 @@
+{
+  "FakeFHIRProviderConfigurations": {
```

Comment:
Rename this file to appsetting.json
Add a new file appsetting.Acceptance.json and add a git ignore rule for it.